swagger/system.yaml

```diff
@@ -0,0 +1,26 @@
+System:
```

Comment:
Are we planning on splitting each set of endpoint's objects out similar to how handlers are?

Reply:
I don't think that's a bad way to approach it, but I wasn't trying to be prescriptive here either.